client/project_update.go

```diff
@@ -9,23 +9,30 @@ import (
 	"github.com/hashicorp/terraform-plugin-log/tflog"
 )
 
+type PasswordProtectionRequest struct {
+	DeploymentType string `json:"deploymentType"`
+	Password       string `json:"password"`
+}
+
 // UpdateProjectRequest defines the possible fields that can be updated within a vercel project.
 // note that the values are all pointers, with many containing `omitempty` for serialisation.
 // This is because the Vercel API behaves in the following manner:
 // - a provided field will be updated
-// - setting the field to an empty value (e.g. ”) will remove the setting for that field.
+// - setting the field to an empty value (e.g. ”") will remove the setting for that field.
```

Comment:
are the different double quotations intended?

Reply:
no, lol.